lib/page.js

```diff
@@ -298,7 +299,8 @@ var Page = function( page_path, options ){
         scripts: '<script src="/compiled/scripts.js"></script>',
         styles: '<link rel="stylesheet" href="/compiled/styles.css" />'
       },
-      layout: this.getLayout()
+      layout: this.getLayout(),
+      isPreview: req.query.preview?true:false
```

Comment:
The more "standard" way to turn a value into a boolean is `!!req.query.preview`.